File: 1903-largest-odd-number-in-string/1903-largest-odd-number-in-string.cpp
class Solution {
public:
    string largestOddNumber(string num) {
        string possible="";
    string out="";
    for(int i=0;i<num.size();i++)
    {
        int x=num[i]-'0';
        possible+=num[i];
        if(x%2!=0)
        {
            out=possible;
        }
    }
    return out;
    }
};

File: 1903-largest-odd-number-in-string/1903-largest-odd-number-in-string.py
class Solution(object):
    def largestOddNumber(self, num):
        """
        :type num: str
        :rtype: str
        """
        out=""
        possible=""
        for i in range(len(num)):
            x=int(num[i])
            if x%2 !=0:
                possible+=num[i]
                out=possible
            else:
                possible+=num[i]
        return out
            